.github/workflows/release_mac.yml

```diff
@@ -23,14 +29,14 @@ jobs:
       # fail-fast: Default is true, switch to false to allow one platform to fail and still run others
       fail-fast: false
       matrix:
-        macos_dev_target: [13.0]
+        macos_dev_target: [12.1, 13.0]
         include:
-#        - macos_dev_target: 12.1
-#          os: macos-12
-#          allow_failure: false
-#          arch: x86_64
-#          python-arch: x64
-#          pretty: "Mac x64"
+        - macos_dev_target: 12.1
+          os: macos-13
+          allow_failure: false
+          arch: x86_64
+          python-arch: x64
+          pretty: "Mac x64"
```

Comment:
Revert the commenting out from @Myoldmopar . hopefully I caught everything that needed reverting...

Reply:
I'm sure it's fine.